lint-staged.config.js

```diff
@@ -2,7 +2,8 @@
 
 module.exports = {
   "*.{js,js.flow,ts}": "eslint --fix --report-unused-disable-directives",
-  "*.md": "prettier --write",
+  "*.{md,mdx}": "prettier --write",
```

Comment:
I'm pretty sure that our Prettier ESLint integration should apply to MDX, too. Try making a Prettier mistake in an `.mdx` file, then run `eslint --fix` on it. If it doesn't work correctly, you might need to set `parser` to `"markdown"` in the `prettier/prettier` rule for `*.mdx` files.
```suggestion
  "*.md": "prettier --write",
```

Reply:
Actually, `parser` should be `"mdx"`, not `"markdown"`. Hopefully that happens automatically, though. 🤞

Reply:
It worked from the beginning. Only reason to avoid it was as below, not being sure about fixes on remark rules. The parser seemed automatic.